content/docs/setup/kubernetes/upgrading-istio/index.md

```diff
@@ -17,22 +17,29 @@ In the following steps, we assume that the Istio components are installed and up
 1. [Download the new Istio release](/docs/setup/kubernetes/download-release/)
 and change directory to the new release directory.
 
-1. Upgrade Istio's [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions)
-via `kubectl apply`, and wait a few seconds for the CRDs to be committed to the Kubernetes API server:
-
-{{< text bash >}}
-$ kubectl apply -f @install/kubernetes/helm/istio/templates/crds.yaml@
-{{< /text >}}
-
 ### Control plane upgrade
 
-The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Policy, Telemetry and
+The Istio control plane components include: Citadel, Ingress gateway, Egress gateway, Pilot, Galley, Policy, Telemetry and
 Sidecar injector.
 
 #### Helm upgrade
 
 If you installed Istio with [Helm](/docs/setup/kubernetes/helm-install/#option-2-install-with-helm-and-tiller-via-helm-install) the preferred upgrade option is to let Helm take care of the upgrade:
 
+1. Upgrade the `istio-init` chart or install new release if it doesn't exists to keep all the Istio's [Custom Resource Definitions or CRDs for short](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) up-to-date:
+
+{{< text bash >}}
+$ helm upgrade --install istio-init install/kubernetes/helm/istio-init --namespace istio-system
+{{< /text >}}
+
+1. Verify all the Istio's CRDs have been committed to the Kubernetes API server by checking all the CRD creation jobs complete with success:
```

Comment:
This sentence is hard to understand, do you mean:
```suggestion
1. Check that all the CRD creation jobs completed successfully to verify that the Kubernetes API server received all the CRDs:
```

Reply:
done